apps/roam/.env.productiondb.example

```diff
@@ -0,0 +1,7 @@
+# BLOB_READ_WRITE_TOKEN =
+# SUPABASE_WORKDIR=/full/path/to/packages/database
+SUPABASE_URL=https://zytfjzqyijgagqxrzbmz.supabase.co
+SUPABASE_PROJECT_ID=zytfjzqyijgagqxrzbmz
+POSTGRES_URL=postgresql://postgres:postgres@zytfjzqyijgagqxrzbmz.supabase.co:54322/postgres
+SUPABASE_JWT_SECRET=gP953JuQgUdzFlO8/3rBTNLraQwS4zItimbZSRPmXvAW2LXa6BMtS/bauTNJDZSbXWVriNuSLFmpCxR/zlVNzg
+SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Inp5dGZqenF5aWpnYWdxeHJ6Ym16Iiwicm9sZSI6ImFub24iLCJpYXQiOjE3NTE1NTIyNTAsImV4cCI6MjA2NzEyODI1MH0.q6fLysc1drLEQ6BXFepL_82xFV3Kknc66u-PC6p4P-o
```

Comment:
Are these OK being in a public repo?

Reply:
OUCH. The postgresURL is surprisingly not dangerous but should not be there. The JWT was a horrible mistake.
Regenerating, will restart the project, will be off for a few minutes.

Reply:
new JWT and tokens.